src/olympia/blocklist/tests/test_mlbf.py

```diff
@@ -36,24 +36,24 @@ def setUp(self):
                 file_kw={'is_signed': True, 'is_webextension': True}),
             updated_by=user_factory())
         # five versions, but only two within block (123.40, 123.5)
-        self.five_ver = Block.objects.create(
+        self.five_ver_block = Block.objects.create(
```

Comment:
most of the changes in this file is code churn because `five_ver` was ambiguously named vs. `five_ver_123_45`